migrations/v8.js

```diff
@@ -1,5 +1,22 @@
 'use strict';
 
+var Reference = require('../reference/v8');
+var ColorOps = require('color-ops');
```

Comment:
No need to support color-ops in the migration. These weren't available in v7 and there are no v8 styles in the wild that would need to be migrated.